app/views/lessons/installation_lessons/index.html.erb

```diff
@@ -1,14 +1,21 @@
-<%= title('Installation Lessons Appendix') %>
+<%= title('Installation Guides') %>
 
-<div class="page-container h-screen">
-  <h1 class="page-heading-title text-center">Installation Lessons Appendix</h1>
-  <ul>
+<div class="page-container min-h-screen">
+  <h1 class="page-heading-title text-center">Installation Guides</h1>
+  <section class="max-w-xl mx-auto space-y-5">
     <% @lessons.each do |lesson| %>
-       <li>
-          <p class="text-1xl text-center m-3 text-gray-600 dark:text-gray-300">
-            <%= link_to lesson.title, lesson_path(lesson), class: 'underline hover:no-underline hover:text-gray-500 dark:hover:text-gray-400' %>
-          </p>
-        </li>
+      <%= render CardComponent.new(classes: "transistion dark:hover:bg-gray-700 hover:bg-gray-100") do |card| %>
```

Comment:
Incredibly minor, but there's a linting issue here that is causing the CI tests to fail. Easy fix, just need to swap the double quotes for single quotes.

You can click on these links in the PR to see the details:

![image](https://user-images.githubusercontent.com/58506115/216848195-90969f82-a016-4f9b-9db8-fc91abceb5c9.png)

and you can run `bin/test` from the app root locally to run all the linters and tests that are used in CI

Reply:
Thanks for letting me know, @ChargrilledChook. It's passing all tests now.